storage/src/state/ledger.rs

```diff
@@ -476,6 +476,7 @@ impl<N: Network> LedgerState<N> {
             .latest_block_hashes_and_headers
             .read()
             .asc_iter()
+            .skip_while(|(_, header)| header.height() == 0) // Skip the genesis block.
```

Comment:
```suggestion
            .filter(|(_, header)| header.height() != 0) // Skip the genesis block.
```
otherwise this will not skip the genesis block unless it's the very first one is the iterator, which is not certain